builder/qemu/builder.go

```diff
@@ -22,10 +21,10 @@ import (
 const BuilderId = "transcend.qemu"
 
 var accels = map[string]struct{}{
-	"none": struct{}{},
-	"kvm":  struct{}{},
-	"tcg":  struct{}{},
-	"xen":  struct{}{},
+	"none": {},
+	"kvm":  {},
+	"tcg":  {},
+	"xen":  {},
```

Comment:
Yeah I think `go imports` / `gofmt` may remove this. I've had to add this type of thing back in manually a few times.